lib/mysql.js

```diff
@@ -459,6 +459,16 @@ MySQL.prototype.ping = function(cb) {
   this.execute('SELECT 1 AS result', cb);
 };
 
+MySQL.prototype.buildExpression = function(columnName, operator, columnValue,
+    propertyValue) {
+  if (operator === 'regexp')
+    return new ParameterizedSQL(columnName + ' REGEXP ?', [columnValue.source]);
```

Comment:
Ignoring the `i` flag silently is not a good idea. We should either report the error or at least print out a warning.